Tools/TestWebKitAPI/Tests/WebCore/ApplicationManifestParser.cpp

```diff
@@ -36,6 +36,18 @@
 using namespace WebCore;
 
 namespace WebCore {
+static inline std::ostream& operator<<(std::ostream& os, const ApplicationManifest::Direction& direction)
+{
+    switch (direction) {
```

Comment:
Can put `using enum ApplicationManifest::Direction` at the top of the function and make the entire thing more terse without losing clarity.